src/test/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/AdapterSDKTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.fileanalysissuite.adaptersdk.impls.jaxrs;
+
+import io.github.fileanalysissuite.adaptersdk.interfaces.extensibility.RepositorySettingDefinition;
+import io.github.fileanalysissuite.adaptersdk.convenience.ConvenientAdapterDescriptor;
+import io.github.fileanalysissuite.adaptersdk.convenience.ConvenientRepositorySettingDefinition;
+import io.github.fileanalysissuite.adaptersdk.interfaces.extensibility.AdapterDescriptor;
+import io.github.fileanalysissuite.adaptersdk.interfaces.framework.TypeCode;
+import io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.serverstubs.model.ItemMetadata;
+import io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.serverstubs.model.RepositoryItem;
+import io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.serverstubs.model.RetrieveFileListRequest;
+import io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.serverstubs.model.RepositoryProperties;
+import io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.serverstubs.model.RetrieveFileDataRequest;
+
+import org.junit.Test;
+import org.junit.BeforeClass;
+import static org.junit.Assert.assertTrue;
+import org.mockito.Mockito;
+import static org.mockito.ArgumentMatchers.any;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonArray;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+public class AdapterSDKTest extends AdapterSDKContainer
+{
+    public static AdapterDescriptor adapterDescriptor;
+    public static RetrieveFileListRequest retrieveFileListRequest;
+    public static RetrieveFileDataRequest retrieveFileDataRequest;
+
+    @BeforeClass
+    public static void setup()
+    {
+        final RepositorySettingDefinition repositorySettingDefinition
+            = ConvenientRepositorySettingDefinition.create("MARK", TypeCode.TEXT, true, true);
+
+        adapterDescriptor = ConvenientAdapterDescriptor.create("TEST", repositorySettingDefinition);
```

Comment:
I used these [Convenient Objects](https://github.com/FileAnalysisSuite/adaptersdk-convenience/tree/main/src/main/java/io/github/fileanalysissuite/adaptersdk/convenience) I found in `adaptersdk-convenience` as they allowed me to test the values in the `testAdapterDescriptorGet` response, unlike with mocking.